Source/WebCore/dom/ViewportArguments.cpp

```diff
@@ -395,6 +410,8 @@ void setViewportFeature(ViewportArguments& arguments, StringView key, StringView
         arguments.shrinkToFit = findBooleanValue(key, value, internalErrorHandler);
     else if (equalLettersIgnoringASCIICase(key, "viewport-fit"_s))
         arguments.viewportFit = parseViewportFitValue(key, value, internalErrorHandler);
+    else if (equalLettersIgnoringASCIICase(key, "interactive-widget"_s))
```

Comment:
Just realized this change does not consult the feature flag. None of this logic should run when the feature is off.

You can:
- Add a `const Document&` parameter to this method, and pass it in from the caller
- Then, check `document.settings().metaViewportInteractiveWidgetEnabled()` here